### DISCUSSION TITLE
About the icon table

### DISCUSSION THREAD

**Discussion by euu2021** [freeplane/freeplane#401] [General]
I'm starting to use the icon table, so I created a couple of tickets with things that I missed in comparison with the other icon input methods (lateral panel and command search), that I used before. 

There is something that is not of my personal interest, so I will just mention here: the icon to toggle that new feature of deleting icons is not there, yet. I say it's not of my personal interest because I keep it activated all the time, as it is so useful.

Another comment, as there was a recent optimization related to icons: I find it curious that searching for icons in the icon table is a little bit slow. In the command search, the search is absolutely instantaneous.

BR,

**Comment by euu2021** [comment]
I will show my use case for icons, so it gets more clear why I'm interested in the icon table (I know that Dimitry likes to see use cases):

When I want to understand a topic, I read several sources of information and collect relevant quotes. As my comprehension grows, I rearrange the branches to reflect my overall view of the topic. The problem of this approach is scalability: after some time I have several pages worth of content, which makes it hard to insert the quotes on proper places and rearrange nodes and branches. Even in Freeplane this scalability problem happens.

I solved this problem using the [node minimizer script](https://sourceforge.net/p/freeplane/discussion/758437/thread/e9200db6c3/#01b5) created by macmarrun. Using it, the map gets very compact, and information dense. For example, the content of this branch is 15 pages long (the nodes are long quotes, as shown in the tooltip example):
![image](https://user-images.githubusercontent.com/77707706/170892367-d799ab95-a3bf-4505-960e-282e10573993.png)

The minimized nodes give me enough cue to allow me to remember what it's about, so I can keep this zoomed out view while I go through the reading of the sources. That's where the icons are useful. I use them to more easily identify parts of the map (styles and color coding also help, and that's why I'm interested in styles, too). So, it's not rare that I need to choose a new icon that more closely resembles the meaning of an area of the map. I was doing that by opening the Command Search, typing "emoji", and scrolling through the options. But, now I noticed that the icon table is a better option, because it shows many icons at once:

![image](https://user-images.githubusercontent.com/77707706/170892827-923998e0-0cb3-431e-9365-38ffca5fe320.png)

**Reply by dpolivaev** [reply]
Thanks for sharing. Exact knowing of your use case does not only allow me to find solutions that can support it better. It also allows me to better see the meaning of my work and life.
